code/modules/surgery/organs/organ.dm

```diff
@@ -43,17 +44,13 @@ var/list/organ_cache = list()
 				owner.organs -= src
 			if((owner.organs_by_name) && (src in owner.organs_by_name))
 				owner.organs_by_name -= src
+*/
```

Comment:
Probably want to replace this commented-out block with a call to Remove() or something similar